.code-samples.meilisearch.yaml

```diff
@@ -8,7 +8,9 @@ get_one_index_1: |-
 list_all_indexes_1: |-
   client.indexes
 create_an_index_1: |-
-  client.create_index('movies', primary_key: 'movie_id')
+  client.create_index('movies', primary_key: 'id')
+document_guide_create_index_primary_key: |-
+  client.create_index('movies', primary_key: 'reference_number')
```

Comment:
(suggestion does not seem to work). It is `primary_key: 'id'` not `reference_number`

Reply:
are you sure about that?

<img width="470" alt="image" src="https://user-images.githubusercontent.com/4116980/163033281-b93b526c-1e52-492f-8fd6-91582f17788a.png">

> https://github.com/meilisearch/documentation/blob/master/.code-samples.meilisearch.yaml

Reply:
my8 bad I was confused in the order of the samples!